packages/api/endpoints/granules.js

```diff
@@ -136,13 +136,10 @@ const update = async (req, res) => {
     updatedBody = restOfBody;
     message = ' Skipped setting status to queued because granule was not running';
   }
-  const now = Date.now();
-  const updatedGranule = {
-    ...existingGranule,
-    updatedAt: now,
-    timestamp: now,
-    ...updatedBody,
-  };
+
+  // eslint-disable-next-line no-unused-vars
+  const { updatedAt, timestamp, ...existingGranuleWithoutTimes } = existingGranule;
```

Comment:
Hmm, re: telling lint to ignore this, it favors terse over linting rules, and that one has somewhat good reasons for existing.  

If the goal here is to just remove timestamps from the object, why not just use our codebase's commonly-used `omit` pattern instead of a babel plugin and a lint override?

Reply:
haven't run into the omit pattern, and this was what made sense. That aside, the destructuring assignment is a useful tool to have even if you don't like this particular use of it.